guide/topics/discordjs/firstCommand.md

```diff
@@ -0,0 +1,163 @@
+authors:
+- NorteX (NorteX-dev)
+categories:
+JavaScript
+tags:
+- Discord.js
+- Bot Developement
+- Basics
+---
+
+Disclaimer: this tutorial assumes you've followed the [Getting Started](gettingStarted.md) tutorial.
+
+## Setup
+In the previous tutorial ([Getting Started](gettingStarted.md)) you learned how to install Node with npm, your programming environment (IDE) and initialized the project.
+
+Just to remind us, this is the code we had set up in the end:
+```javascript
+// Import the "Client" class from discord.js library
+const { Client } = require("discord.js");
+// Initialize the client and assign it to a constant variable "client"
+const client = new Client();
+// Listen to the "ready" event and print "I am ready!" whenever the bot is ready.
+client.once("ready", () => console.log("I am ready!"));
+// Authenticate the bot with your token from the Discord Developer Page
+client.login("-- Your Token --");
+```
+Upon running above code, the bot printed "I am ready!" in the console. Great!
+
+Now, that we are up to date with the previous tutorial, we can start writing more code.
+
+## Creating the event
+First off, you have to understand that commands are in fact just messages with a specific prefix.
+That being said, to create commands for your bot, you'll have to listen to the `message` event.
+
+Listening to the `message` event is very similar to listening to the `ready` event and is as follows:
+```javascript
+client.on("message", (message) => {
+  console.log("Hello!");
+});
+```
+Let's brake this code down. As with the `ready` event, we are accessing the `client` variable. Then, we are using `.on()` which is a method that creates an _event listener_ on our Discord bot client.
+
+But wait! We can notice, that in the `ready` event, the method is `.once`, not `.on`!
+
+The difference between these two, is `once` - as the name suggests - only invokes once per execution and cannot be run again. `on` on the other hand can execute however many times the event happens. For our `message` event, we want to use the `on` method, as the command is sent more than one time - unlike `ready`, which should only execute once.
+
+Now with the `once` and `on` difference out of the way, we can look at the arguments - for both methods, it will always be two:
+- Event name
+- A callback function
+
+The event name is obviously the name of the event that we want to listen to. There are many events, e.g. these which we have learnt already: `ready` and `message`, but also `guildCreate`, `guildMemberAdd`, `messageDelete`, `roleCreate` and many, many more. You can see all of them in this [Discord events cheatsheet by koad](https://gist.github.com/koad/316b265a91d933fd1b62dddfcc3ff584).
+
+The other parameter, a function, is the function that will be invoked when the event is fired. Oftentimes, it takes in parameters specific for that type of event. Our `message` event, only returns 1 parameter: a message object which we will be able to utilize in a second.
```

Comment:
> The other parameter, a function, is the function that will be....
That's a bit hard to read IMO. I suggest being more straight forward.
"The second parameter is a call-back. This will be executed once the event is triggered."